openid-caep-interoperability-profile-1_0.md

```diff
@@ -0,0 +1,193 @@
+---
+title: CAEP Interoperability Profile 1.0 - draft 01
+abbrev: caep-interop
+docname: caep-interoperability-profile-1_0
+date: 2023-11-17
+
+ipr: none
+cat: std
+wg: Shared Signals
+
+coding: us-ascii
+pi:
+  toc: yes
+  sortrefs: yes
+  symrefs: yes
+  private: yes
+
+author:
+      -
+        ins: A. Tulshibagwale
+        name: Atul Tulshibagwale
+        org: SGNL
+        email: atul@sgnl.ai
+
+contributor:
+      -
+        ins: A. Deshpande
+        name: Apoorva Deshpande
+        org: Okta
+        email: apoorva.deshpande@okta.com
+
+normative:
+  RFC9493: # Subject Identifier Formats for SETs
+  SSF:
+    target: http://openid.net/specs/openid-sse-framework-1_0.html
+    title: OpenID Shared Signals and Events Framework Specification 1.0
+    author:
+      -
+        ins: A. Tulshibagwale
+        name: Atul Tulshibagwale
+        org: Google
+      -
+        ins: T. Cappalli
+        name: Tim Cappalli
+        org: Microsoft
+      -
+        ins: M. Scurtescu
+        name: Marius Scurtescu
+        org: Coinbase
+      -
+        ins: A. Backman
+        name: Annabelle Backman
+        org: Amazon
+      -
+        ins: John Bradley
+        name: John Bradley
+        org: Yubico
+  CAEP:
+    target: https://openid.net/specs/openid-caep-specification-1_0.html
+    title: OpenID Continuous Access Evaluation Profile 1.0
+    author:
+      -
+        ins: T. Cappalli
+        name: Tim Cappalli
+        org: Microsoft
+      -
+        ins: A. Tulshibagwale
+        name: Atul Tulshibagwale
+        org: SGNL
+  RFC8935:
+
+
+--- abstract
+This document defines an interoperability profile for implementations of the Shared Signals Framework (SSF) {{SSF}} and the Continuous Access Evaluation Profile (CAEP) {{CAEP}}. It is organized around use-cases that improve security of authenticated sessions. It specifies certain optional elements from within the SSF and CAEP specifications as being required to be supported in order to be considered as an interoperable implementation.
+
+--- middle
+
+# Introduction {#introduction}
+SSF and CAEP together enable improved session security outcomes. This specification defines the minimum required features from SSF and CAEP that an implementation MUST offer in order to be considered as an interoperable implementation. This document defines specific use cases. An implementation may support only a subset of the use cases defined herein, and SHALL be considered an interoperable implementation for the specific use-cases it supports. The following use-cases are considered as a part of this specification:
+
+Session Revocation
+: A SSF Transmitter or Receiver is able to respectively generate or respond to the CAEP session-revoked event
```

Comment:
```suggestion
: An SSF Transmitter or Receiver is able to respectively generate or respond to the CAEP session-revoked event
```

Reply:
I'm generally using "a SSF" because it's "a Shared Signals Framework"... I think both can be argued to be correct.